pkg/logs/internal/launchers/file/launcher.go

```diff
@@ -38,9 +38,9 @@ var ContainersLogsDir = "/var/log/containers"
 // DefaultSleepDuration represents the amount of time the tailer waits before reading new data when no data is received
 const DefaultSleepDuration = 1 * time.Second
 
-// Scanner checks all files provided by fileProvider and create new tailers
+// Launcher checks all files provided by fileProvider and create new tailers
 // or update the old ones if needed
-type Scanner struct {
+type Launcher struct {
```

Comment:
I agree that file `Launcher` is a better name given it's use case. 
We will have to update docs including the readme you also have a PR open (and probably the more recent slide decks) for to account for this for consistency.

Reply:
I merged that README here and updated it, so landing those in the correct order should work.  I'm not terribly comfortable editing other people's presentations, though.